src/interfaces/fees.ts

```diff
@@ -6,18 +6,26 @@ import { PROTOCOL_FEES } from "../services/subgraph/queries";
 import { FeesResponse } from "../services/subgraph/responses";
 import { Usdc } from "../types";
 
+const BigZero = new BigNumber(0);
+
 export class FeesInterface<C extends ChainId> extends ServiceInterface<C> {
   async protocolFees(since: Date): Promise<Usdc> {
     const response = await this.yearn.services.subgraph.fetchQuery<FeesResponse>(PROTOCOL_FEES, {
       sinceDate: since.getTime().toString(),
     });
 
+    let result = BigZero;
+
+    if (!response?.data || !response.data?.transfers) {
+      return result.toFixed(0);
+    }
+
     const transfers = response.data.transfers;
 
     return transfers
```

Comment:
nit: We could destructure `tokenAmountUsdc` to make it a bit more readable
```suggestion
    return transfers.reduce((prev, { tokenAmountUsdc }) => prev.plus(toBN(tokenAmountUsdc || 0)), result).toFixed(0);
```